src/services/impressions/dto.js

```diff
@@ -36,6 +36,10 @@ module.exports = {
           if (entry.label) keyImpression.label = entry.label;
           if (entry.bucketingKey) keyImpression.bucketingKey = entry.bucketingKey;
 
+          properties = {};
+          if (FS) properties.full_story_url = FS.getCurrentSessionURL();
```

Comment:
Ideally we would add a check against the configuration if full story integration is enabled